pkg/morph/client/notary.go

```diff
@@ -48,12 +48,10 @@ const (
 	setDesignateMethod    = "designateAsRole"
 
 	notaryBalanceErrMsg = "can't fetch notary balance"
+	notaryNotEnabledMsg = "notary support was not enabled on this client"
```

Comment:
maybe add `Err` or `Panic` word in var name like in the previous var?

Reply:
Fixed.